nx-X11/lib/X11/XlibInt.c

```diff
@@ -409,66 +423,36 @@ _XWaitForWritable(
 	   XXX - what if cv is NULL and someone else comes along after
 	   us while we are waiting?
 	   */
-	   
+
 	if (!dpy->lock ||
 	    (!dpy->lock->event_awaiters &&
 	     (!dpy->lock->reply_awaiters ||
 	      dpy->lock->reply_awaiters->cv == cv)))
 #endif
 
 #ifndef NX_TRANS_SOCKET
-#ifdef USE_POLL
+#  ifdef USE_POLL
 	    filedes.events = POLLIN;
 	filedes.events |= POLLOUT;
-#else
+#  else
 	FD_SET(dpy->fd, &r_mask);
         FD_SET(dpy->fd, &w_mask);
-#endif
+#  endif
 #endif /* #ifndef NX_TRANS_SOCKET */
 
 	do {
-#ifdef NX_TRANS_SOCKET
```

Comment:
I am not sure what happened here, but in my latest commit this code is not missing. Did you look at the wrong commit?